components/script/Cargo.toml

```diff
@@ -21,6 +21,9 @@ path = "../util"
 [dependencies.msg]
 path = "../msg"
 
+[dependencies.net]
+path = "../net"
```

Comment:
Oh look, a @jdm!

You want the whole of the HOSTS_FILE stuff moved into net_traits then?